### PR TITLE
Update speakers-page.html

### PR DESCRIPTION
**Pull request by pentium10**

### CODE REVIEW

File: src/pages/speakers-page.html [hunk 1]
```diff
@@ -150,8 +150,8 @@
     </style>
 
     <polymer-helmet
-      title="Speakers | GDG DevFest Ukraine"
-      description="Learn more about speakers"
+      title="{$ heroSettings.speakers.title $} | {$ title $}"
+      description="112{$ heroSettings.speakers.metaDescription $}"
```
Comment (ozasadnyy) [inline src/pages/speakers-page.html]: Please remove `112` and ready to go :)

Comment (sophieH29) [inline src/pages/speakers-page.html]: what is the number at the beginning ```112```?

Comment (ozasadnyy) [reply src/pages/speakers-page.html]: @pentium10 we still waiting for your changes :)

**Comment by pentium10** [comment]
see now

**Pull request merged**